{application,try_catch_errors,
             [{applications,[kernel,stdlib,elixir,logger]},
              {description,"try_catch_errors"},
              {modules,['Elixir.MyError','Elixir.RunAfter',
                        'Elixir.TryCatchErrors']},
              {registered,[]},
              {vsn,"0.1.0"}]}.
